index.js

```diff
@@ -75,7 +75,7 @@ class NewRelic {
     }
     const route = currentState.routes[currentState.index];
     if (route.routes) {
-      return getActiveRouteName(route);
+      return getCurrentRouteName(route);
```

Comment:
i think you should be add `this.getCurrentRouteName`

Reply:
i will take care of this.